From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Tue, 7 May 2024 15:20:46 +0000
Subject: [PATCH] Revert "[Fixit] Update Password Manager to Google Password
 Manager"

This reverts commit 92cbf22d55e8fdc8b54155823aa94bec3bcb5ad3.
---
 chrome/browser/ui/android/strings/android_chrome_strings.grd    | 2 +-
 .../IDS_PASSWORD_MANAGER_SETTINGS_TITLE.png.sha1                | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/chrome/browser/ui/android/strings/android_chrome_strings.grd b/chrome/browser/ui/android/strings/android_chrome_strings.grd
index 95ba6fce59df3..ba4e6a91fb92f 100644
--- a/chrome/browser/ui/android/strings/android_chrome_strings.grd
+++ b/chrome/browser/ui/android/strings/android_chrome_strings.grd
@@ -660,7 +660,7 @@ For more settings that use data to improve your Vanadium experience, go to <ph n
         Passwords
       </message>
       <message name="IDS_PASSWORD_MANAGER_SETTINGS_TITLE" desc="Title for the settings menu item leading to the Password Manager UI surface. This allows users to e.g. manage their saved passwords, toggle saving and auto-sign-in on/off, etc. [CHAR_LIMIT=32]">
-        Google Password Manager
+        Password Manager
       </message>
       <message name="IDS_PASSWORD_SAVING_ON_BY_ADMINISTRATOR" desc="Subtitle for the settings menu item leading to the Password Manager UI surface, shown when the device administrator set saving passwords to enabled.">
         Password saving is turned on by your administrator
diff --git a/chrome/browser/ui/android/strings/android_chrome_strings_grd/IDS_PASSWORD_MANAGER_SETTINGS_TITLE.png.sha1 b/chrome/browser/ui/android/strings/android_chrome_strings_grd/IDS_PASSWORD_MANAGER_SETTINGS_TITLE.png.sha1
index 8c85ee94351d5..ca8260bd6162d 100644
--- a/chrome/browser/ui/android/strings/android_chrome_strings_grd/IDS_PASSWORD_MANAGER_SETTINGS_TITLE.png.sha1
+++ b/chrome/browser/ui/android/strings/android_chrome_strings_grd/IDS_PASSWORD_MANAGER_SETTINGS_TITLE.png.sha1
@@ -1 +1 @@
-863c9a14b01d82c6c3aa24757a6bea80341db98f
\ No newline at end of file
+a9b040f33f109e043b69de841ecd5123fc97f97b
\ No newline at end of file
